ui/jobs/jobs.js

```diff
@@ -1152,7 +1152,7 @@ class Bars {
     if (oath < 50) {
       this.o.oathTextBox.classList.add('low');
       this.o.oathTextBox.classList.remove('mid');
-    } else if (oath < 100) {
+    } else if (oath < 90) {
```

Comment:
Did you mean to do this for pld? Or it seems like maybe you'd want to do this for both pld and drk?

Reply:
My fault, was meant for DRK. It was too early and I am only a student!